index.html

```diff
@@ -24,19 +27,159 @@ <h5>CB</h5>
+            <h2>Multi-Post Stories</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <button type="button" class="seeProject">See Project</button>
+        </div>
+    </section>
+    <section class="About Myself">
+        <h1>About<br> Myself</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t
+            hestiate to contact me.
+        </p>
+        <h4 class="connect">LET'S CONNECT</h4>
+        <ul class="second socials">
+            <li><a href="https://github.com/Christianib003"><i class="fab fa-github"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226"><i
+                        class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-instagram"></i></a></li>
         </ul>
+        <button type="button" class="Resume">Get my resume</button>
+        <section class="lang-details">
+            <ul class="lang-list">
+                <li class="lang-one">
+                    <div class="drop-list">
+                        <h3>Languages</h3>
+                        <span><i class="arrow arrow-down"></i></span>
+                    </div>
+                    <ul class="know">
+                        <li class="know-lang">
+                            <div class="image-container">
+                                <img src="./Images/JS-icon.png" class="" alt="javascript">
+                            </div>
+                            <p class="know-pa">JavaScript</p>
+                        </li>
+                        <li class="know-lang">
+                            <div class="image-container">
+                                <img src="./Images/HTML-icon.png" class="" alt="HTML">
+                            </div>
+                            <p class="know-pa">HTML</p>
+                        </li>
+                        <li class="know-lang">
+                            <div class="image-container">
+                                <img src="./Images/CSS-icon.png" class="" alt="css">
+                            </div>
+                            <p class="know-pa">CSS</p>
+                        </li>
```

Comment:
- [ ] Kindly make sure your design matches the original Figma design as much as possible for a better user interface😎
- Your design | Figma design
![screenshot (18)](https://user-images.githubusercontent.com/44624138/149627709-b9ef0d5e-01fc-4974-ae83-70605971f22b.png) | ![screenshot (19)](https://user-images.githubusercontent.com/44624138/149627716-fd602a72-41ed-489c-ade6-06f6b6d41489.png)